src/utils/jwt/jwtUtils.js

```diff
@@ -0,0 +1,101 @@
+import {validateSchema} from "../schemas/validateSchema";
```

Comment:
Ogólnie niektóre z tych funkcji mogłyby być prywatne - dodaj przed nimi podkreślnik, to będzie to taka trochę konwencja